suse-module-tools.spec

```diff
@@ -99,6 +101,16 @@ This package contains the legacy "weak-modules" script for kernel
 module package (KMP) support. It was replaced by "weak-modules2" in
 SLE 11 and later. It is still used by the DKMS module packaging framework.
 
+%if 0%{?suse_version} >= 1550
+%package -n suse-kernel-rpm-scriptlets
+Summary:        Traditional kernel scriptlets
+Requires:       suse-module-tools = %{version}
+Provides:       suse-module-tools:/usr/lib/module-init-tools/kernel-scriptlets
```

Comment:
What exactly does this syntax mean?

Reply:
it's a split-provides: https://en.opensuse.org/openSUSE:Package_dependencies#Splitting_off_a_sub-package